Sources/SKSupport/dlopen.swift

```diff
@@ -60,32 +73,53 @@ public struct DLOpenFlags: RawRepresentable, OptionSet {
   }
 }
 
+#if os(Windows)
+public enum DLError: Swift.Error {
+  case LoadLibrary(String)
+  case FreeLibrary(String)
+}
+#else
```

Comment:
Let's just unify these since they're the same other than the name. How about `DLError.open` and `DLError.close`?

Reply:
Seems reasonable. They're now open and close.